_posts/2020-01-19-zarzadzanie-zadaniami.md

```diff
@@ -0,0 +1,288 @@
+6. dedykowana aplikacja
+  * lista zadań w prywatnej “chmurze”
+  * możliwość wprowadzenia struktury/etykiet
+  * dostępność z internetu
+  * aplikacja mobilna
+  * brak narzuconego/wymaganego workflow
+
+Powyższa lista żadną miarą nie jest kompletna. Jestem przekonany, że istnieje jeszcze sporo ciekawych rozwiązań. Korzystanie z dedykowanej aplikacji jest jednak tak wygodne, że przestałem szukać dalej.
+
+## Moje narzędzie
+<!-- specjalizowana apka -->
+
+Ja do zarządzania zadaniami wykorzystuję aplikację [Todoist](httpw://todoist.com). Mój wybór ułatwiły następujące właściwości aplikacji:
+1. łatwość dostępu
+  * WWW
+  * dedykowana aplikacja mobilna
+  * aplikacja Chrome (np. uruchamianie w Ubuntu jako osobne okno które mogę przykleić do pulpitu)
+  * rozszerzenie Chrome (np. do szybkiego dodawania zadań)
+2. brak narzuconego sposobu zarządzania
+  * nie muszę definiować statusów/workflow zadań
+  * mogę dowolnie interpretować priorytety/etykiety/projekty itp.
+3. dodatkowe wbudowane funkcjonalności
+  * etykiety
+  * projekty
+  * **filtry**
+  * przypomnienia czasowe i geograficzne
+  * integracje (email, Gmail, Chrome, Asystent Google, Slack, Kalendarz Google, IFFT...)
+4. aspekt społecznościowy
+  * sporo materiałów opisujących podejścia różnych osób do zarzadzania zadaniami w Todoist
+  * Youtube, Twitter, blog
+
+Dodawanie zadań jest proste. Wymaga minimalnej wiedzy o zadaniu jednocześnie umożliwia ustawienie wszystkich aspektów natychmiast. Dzięki integracjom mogę dodawać zadanie w najwygodniejszy aktualnie sposów, np.
+* jak prowadzę samochód mogę powiedzieć do Asystenta Google: "notatka dla siebie sprawdź jutro czy poszła faktura za X"
+* jak mogę pisać na telefonie mogę dodać zadnie dotykając jeden przycisk na pulpicie telefonu i wpisać treść zadania na klawiaturze
+  * etykiety, terminy, priorytety są zaczytywane z tekstu
+  * można też je łatwo wybrać przyciskami
+* mogę dodać zadanie wysyłając email na dedykowany adres
+  * bardzo przydatne jako forward - stworzenie zadania z maila zajmuje 4 sekundy
+* mogę dodać zadanie z menu kontekstowego w przeglądarce
+* mogę dodać zadanie z poziomu rozmowy na Slack wpisująć "/todoist sprawdź kiedy grają Gwiezdne Wojny"
+* mogę dodać zadanie bezpośrednio w aplikacji
+* mogę automatycznie tworzyć zadania w odpowiedzi na zdarzenia IFFT
+
+Przeglądanie zadań jest proste. Zadania bez wskazanego projektu wpadają do skrzynki odbiorczej. Jest to worek na wszystko co wpadło (o ile podczas tworzenia zadania nie podałem wprost projektu). Przeglądając skrzynkę odbiorczą mogę:
+* uzupełnić opis zadania
+  * notatki w formie tekstu i załączników
+* dodać podzadania
+* ustawić priorytet
+  * 5 poziomów
+  * steruje kolejnością wyświetlania zadań (prosto, bez ręcznego sortowania i nadmiernego wyboru)
+* przypisać do projektu
+  * pozwala zdefiniować osobne worki zadań
+  * ułatwia _"ogarnięcie"_ dużej ilości zadań
+* ustawić etykietę
+  * wykorzystuję jako wskazówka czego potrzebuję do wykonania zadania
```

Comment:
w jakiej osobie jest to wymieniane? Jeżeli w pierwszej to powinno być "steruję", a jeżeli w trzeciej to wówczas wykorzystuje

osobna sprawa to wykorzystuję jako wskazówk**ę,** czego potrzebuję do wykonania zadania